app/models/physical_rack.rb

```diff
@@ -0,0 +1,6 @@
+class PhysicalRack < ApplicationRecord
+  belongs_to :ext_management_system, :foreign_key => :ems_id,
+             :class_name => "ManageIQ::Providers::PhysicalInfraManager", :inverse_of => :physical_racks
+
+  has_many :physical_servers, :dependent => :destroy, :inverse_of => :physical_rack
```

Comment:
Actually for this one I think you want it to be dependent => :nullify because physical_servers is a top-level collection and deletion will be done by ems_refresh